frontend/src/components/shorts/ShortsPlayer.tsx

```
@@ -11,137 +11,63 @@ interface ShortsPlayerProps {
+            setCurrentIndex(Number((entry.target as HTMLElement).dataset.index));
           }
-        });
+        }
       },
-      {
-        threshold: 0.5,
-        root: containerRef.current
-      }
+      { threshold: 0.5, root: containerRef.current }
     );
+    requestAnimationFrame(() => slideRefs.current.forEach((s) => observer.observe(s)));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)